.github/scripts/windows/find-target-branch.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+
+findstr /m "PHP_VERSION_ID 800" %~dp0..\..\..\main\php_version.h >nul 2>&1
+if %errorlevel% equ 0 (
+    set BRANCH=8.0
+    goto :eof
+)
+findstr /m "PHP_VERSION_ID 810" %~dp0..\..\..\main\php_version.h >nul 2>&1
+if %errorlevel% equ 0 (
+    set BRANCH=8.1
+    goto :eof
+)
+set BRANCH=master
```

Comment:
Even if I still think that [we should get of our own branch detection in the long run](https://github.com/php/php-src/commit/a4179e4c92b6365d39e09cb9cd63c476848013af#r72424332), I suggest to rewrite this to cater to arbitrary branches (otherwise we'd need to add code for 8.2 pretty soon).

Something like the following should do (maybe add some error handling):
````bat
@echo off
for /f "usebackq tokens=3" %%i in (`findstr PHP_MAJOR_VERSION main\php_version.h`) do set BRANCH=%%i
for /f "usebackq tokens=3" %%i in (`findstr PHP_MINOR_VERSION main\php_version.h`) do set BRANCH=%BRANCH%.%%i
````

Reply:
once all tests pass, we can do it

what about `master`?

Reply:
Ah, good catch! I think we don't need special treatment for "master", if we don't pass the branch to `phpsdsk_deps`. For `DEPS_DIR`, `8.2` should be fine, and `phpsdk_deps` should [find out that the branch is actually "master"](https://github.com/php/php-sdk-binary-tools/blob/6ee0f9557b5874b7010513785f399c8745bc9dc1/lib/php/libsdk/SDK/Config.php#L230-L241).

Reply:
I applied your code for branch detection and it works well 👍